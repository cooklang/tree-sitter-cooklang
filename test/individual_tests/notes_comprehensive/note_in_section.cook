= Cooking Tips

> Always taste as you go
> Adjust seasoning at the end