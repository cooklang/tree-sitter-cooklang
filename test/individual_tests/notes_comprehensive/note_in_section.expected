(recipe
  (section
    header: (section_header
      name: )
    content: (note)
    content: (note)))